appveyor.yml

```diff
@@ -50,11 +50,11 @@ install:
 - cmd: IF %GTEST%==true IF NOT EXIST KQvKQ.rtbz curl --remote-name-all https://tablebase.lichess.ovh/tables/standard/3-4-5/K{P,N,R,B,Q}vK{P,N,R,B,Q}.rtb{w,z}
 - cmd: cd C:\projects\lc0
 cache:
-  - C:\cache -> appveyor.yml
+  - C:\cache
   - 'C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0'
   - C:\projects\lc0\subprojects\packagecache
 before_build:
-- cmd: git submodule update --init --recursive
+- cmd: git submodule update --init --recursive --remote
```

Comment:
Hm, I think this will fetch the latest version of from the remote.
I think we need to fetch the one which is listed as "current" (I don't know how that works exactly with submodules).

Reply:
I did that with an older tree where the submodule update was not committed. It is indeed unnecessary.